pkgs/by-name/an/ansilove/package.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, libansilove
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ansilove";
+  version = "4.2.0";
+
+  src = fetchFromGitHub {
+    owner = "ansilove";
+    repo = pname;
```

```suggestion
    repo = "ansilove";
```
`repo = pname` is discouraged.